aten/src/ATen/native/quantized/cpu/qnnpack/include/conv_utils.h

```diff
@@ -226,6 +226,12 @@ struct conv_param_t {
         ukernel_type = pytorch_qnnp_ukernel_type_conv;
       }
     }
+
+    if (per_channel && ukernel_type == pytorch_qnnp_ukernel_type_xzp_gemm) {
```

Comment:
The rules of operator precedence are unambiguous, but I personally err on the side of grouping with parentheses explicitly.  This is a personal preference of course.  Please feel free to ignore.